modules/gapi/test/gapi_fluid_test.cpp

```diff
@@ -244,6 +244,45 @@ TEST(Fluid, Sum_2_Mats_And_Scalar)
     EXPECT_EQ(0, cv::countNonZero(out_mat != ref_mat));
 }
 
+TEST(Fluid, EqualizeHist)
+{
+    cv::GMat mat;
+    cv::GArray<int> arr;
+
+    cv::GComputation c(cv::GIn(mat, arr), cv::GOut(TEqualizeHist::on(mat, arr)));
+    cv::Mat in_mat(320, 480, CV_8UC1),
+            out_mat(320, 480, CV_8UC1),
+            ref_mat(320, 480, CV_8UC1);
+
+    cv::randu(in_mat, 200, 240);
+    std::vector<int> in_vec(256);
+
+    // Calculate normalized accumulated integral transformation array for gapi
+    for(int i = 0; i < in_mat.rows; ++i)
+        for(int j = 0; j < in_mat.cols; ++j)
+            ++in_vec[in_mat.at<uint8_t>(i, j)];
```

Comment:
This is just cv::calcHist

Reply:
Note, it should work with a regular std::vector for output

Reply:
It doesn't work with vector and `calcHist` requires a lot variables to setup (convert single parameters into vectors). So, I think those 3 rows of code are much simpler